apps/thundermoon_web/lib/thundermoon_web/live/chat_live/message.ex

```diff
@@ -1,44 +1,30 @@
 defmodule ThundermoonWeb.ChatLive.Message do
-  use ThundermoonWeb, :live_component
+  use Phoenix.Component
 
-  @impl true
-  def update(assigns, socket) do
-    {:ok,
-     socket
-     |> assign(assigns)
-     |> assign_is_current_user
-     |> assign_message_color}
-  end
-
-  @impl true
-  def render(assigns) do
+  def row(assigns) do
     ~H"""
     <div class="flex">
-      <%= if @is_current_user do %>
-        <div class="w-3/12"></div>
-      <% end %>
-      <div class={"#{@message_color} rounded-md px-4 py-2 mb-2 w-9/12"}>
+      <.indent for_current_user={true} current_user={@current_user} message={@message} />
+      <div class={"#{message_color(assigns)} rounded-md px-4 py-2 mb-2 w-9/12"}>
         <div class="font-bold"><%= @message.user %></div>
         <div class="font-light"><%= @message.text %></div>
       </div>
-      <%= unless @is_current_user do %>
-        <div class="w-3/12"></div>
-      <% end %>
+      <.indent for_current_user={false} current_user={@current_user} message={@message} />
```

Comment:
@grrrisu 

I believe it's possible to remove this line.

Secondly, Re-thinking about the concept, this HTML layout could also be a solution:

```html
<div class="flex {{`justify-end` if current_user == message.user}}">
  <div class="w-3/4"> <!-- or "w-9/12" 😉 -->
    <!-- message  -->
  </div>
</div>
```

That means you only need to decide at one point whether to "pull" the message box to the right edge (`justify-end`). In the default cause it stays on the left (aka `justify-start`).

Reply:
You're right, that's way better. 
(I'll need to find a better opportunity to use a local function in a tag 😉)